testing/nginx-naxsi/APKBUILD

```diff
@@ -32,119 +51,153 @@ source="http://nginx.org/download/$_pkgname-$pkgver.tar.gz
-		--http-fastcgi-temp-path=/tmp/$_pkgname/fastcgi \
-		--user=nginx \
-		--group=nginx \
-		--with-http_ssl_module \
-		--with-http_realip_module \
-		--with-http_addition_module \
-		--with-http_sub_module \
-		--with-http_dav_module \
-		--with-http_flv_module \
-		--with-http_mp4_module \
-		--with-http_gunzip_module \
-		--with-http_gzip_static_module \
-		--with-http_random_index_module \
-		--with-http_secure_link_module \
-		--with-http_stub_status_module \
-		--with-http_auth_request_module \
-		--with-threads \
-		--with-stream \
-		--with-stream_ssl_module \
-		--with-http_slice_module \
-		--with-mail \
-		--with-mail_ssl_module \
-		--with-file-aio \
-		--with-http_v2_module \
-		--without-http_uwsgi_module \
-		--without-http_scgi_module \
-		--add-module="$srcdir/naxsi-$_ngx_naxsi_ver/naxsi_src" \
-		--add-module="$srcdir/ngx_cache_purge-$_ngx_cache_purge_ver" \
-		--add-module="$srcdir/nginx-upstream-fair-$_ngx_upstream_fair_ver" \
-		--add-module="$srcdir/nginx-http-sysguard-$_ngx_http_sysguard_ver" \
+                --http-client-body-temp-path=/var/lib/$_pkgname/tmp/client_body \
+                --http-proxy-temp-path=/var/lib/$_pkgname/tmp/proxy \
+                --http-fastcgi-temp-path=/var/lib/$_pkgname/tmp/fastcgi \
+                --with-perl_modules_path=/usr/lib/perl5/vendor_perl \
+                \
+                --user=$pkgusers \
+                --group=$_grp_ngx \
+                --with-threads \
+                --with-file-aio \
+                --without-http_uwsgi_module \
+                --without-http_scgi_module \
+                \
+                --with-http_ssl_module \
+                --with-http_v2_module \
+                --with-http_realip_module \
+                --with-http_addition_module \
+                --with-http_sub_module \
+                --with-http_dav_module \
+                --with-http_flv_module \
+                --with-http_mp4_module \
+                --with-http_gunzip_module \
+                --with-http_gzip_static_module \
+                --with-http_auth_request_module \
+                --with-http_random_index_module \
+                --with-http_secure_link_module \
+                --with-http_slice_module \
+                --with-http_stub_status_module \
+                --with-http_realip_module \
+                --with-http_xslt_module=dynamic \
+                --with-http_image_filter_module=dynamic \
+                --with-http_geoip_module=dynamic \
+                --with-http_perl_module=dynamic \
+                --with-mail=dynamic \
+                --with-mail_ssl_module \
+                --with-stream=dynamic \
+                --with-stream_ssl_module \
+                \
+                --add-dynamic-module="$_ngx_naxsi_dir" \
+                --add-dynamic-module="$_ngx_cache_purge_dir" \
+                --add-dynamic-module="$_ngx_upstream_fair_dir" \
+                --add-dynamic-module="$_ngx_http_sysguard_dir" \
```

Comment:
Please use only tabs for indentation, I really don’t enjoy fixing it all the time. ;)